binding.c

```diff
@@ -397,7 +428,7 @@ bare_os_memory_usage (js_env_t *env, js_callback_info_t *info) {
 
 #define V(name, property) \
   { \
-    if (stats.property != (size_t) -1) { \
+    if (stats.property != (size_t) - 1) { \
```

Comment:
```suggestion
    if (stats.property != (size_t) -1) { \
```

What's your `clang-format --version`?

Reply:
It was `18.1.8`, now upgraded to `19.1.7`